test/ci/kokoro/run_integ_tests.sh

```diff
@@ -42,6 +42,14 @@ if [[ $KOKORO_JOB_NAME =~ "macos" ]]; then
 fi
 
 function latest_python_release {
```

Comment:
Can we change this to "preferred_python_release" to reduce later confusion? There are comments further down clarifying that they're getting the "latest" available, so I think everything will still semantically make sense after the change.

Reply:
Good point! Done.

Reply:
Oops, missed updating the calling function

Reply:
Now it's done.